client/main/tableview.js

```diff
@@ -17,6 +17,8 @@ const Notify = require('./notification');
 const { csvifyCells, htmlifyCells } = require('./utils/formatio');
 const host = require('./host');
 const ActionHub = require('./actionhub');
+const ContextMenus = require('./contextmenu/contextmenus');
+const ContextMenu = require('./contextmenu');
```

Comment:
what if you require `./contextmenu/contextmenus` into `./contextmenu`, and then export `get/createVariableMenuItems()`, etc. from `./contextmenu`. that would simplify these requires.

and perhaps these calls could be made less cumbersome?:

`ContextMenu.show(ContextMenus.getRowMenuItems(), event.clientX, event.clientY);`

not sure. what do you think?